packages/dds/cell/src/cell.ts

```diff
@@ -120,22 +113,10 @@ export class SharedCell<T = any>
 	 * @param id - Unique identifier for the `SharedCell`.
 	 */
 	// eslint-disable-next-line @typescript-eslint/explicit-member-accessibility
-	constructor(
-		id: string,
-		runtime: IFluidDataStoreRuntime,
-		attributes: IChannelAttributes,
-		options?: ICellOptions,
-	) {
+	constructor(id: string, runtime: IFluidDataStoreRuntime, attributes: IChannelAttributes) {
```

Comment:
i missed this in previous PR and can't comment on the line directly, but you should remove the `@alpha` tag from the constructor.

Reply:
Forgot to check this before merging, will fix it in next PR related to SharedCell